### PR TITLE
Added tests for contact form and app compoment and utils

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**

- Added tests for app component
- Added tests for contact form component
- Added tests for utils
- Some minor bug fixes

Please review @hemantjadon

### CODE REVIEW

File: src/app/contact/contact-form/contact-form.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="overlay">
 	<div class="overlay-content">
-		<button class="form-close" (click)="hideContactForm.emit(false)">x</button>
+		<button class="form-close" (click)="hideContactForm.emit(false);">x</button>
```
Comment (hemantjadon) [inline src/app/contact/contact-form/contact-form.component.html]: No need of this semicolon

File: src/app/utils/type.ts [hunk 2]
```diff
@@ -16,6 +16,6 @@ export function actionTypeCheck<T>(label: T | ''): T {
 	}
 
 	typeCache[<string>label] = true;
-
+	console.log(<T>label);
```
Comment (hemantjadon) [inline src/app/utils/type.ts]: Why are you logging this ?

Comment (Achint08) [inline src/app/utils/type.ts]: Done!

**Comment by Achint08** [comment]
@hemantjadon Done!

**Pull request merged**